.github/ISSUE_TEMPLATE/1-feature-gate.yml

```diff
@@ -0,0 +1,70 @@
+name: Feature Gate Tracker
+description: Track the development and status of an on-chain feature
+title: "[Feature Gate]: "
+labels: ["feature-gate"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Steps to add a new feature are outlined below. Note that these steps only cover
+        the process of getting a feature into the core Solana code.
+
+        - For features that are unambiguously good (ie bug fixes), these steps are sufficient.
+
+        - For features that should go up for community vote (ie fee structure changes), more
+        information on the additional steps to follow can be found at:
+        <https://spl.solana.com/feature-proposal#feature-proposal-life-cycle>
+
+        1. Generate a new keypair with `solana-keygen new --outfile feature.json --no-passphrase`
+            - Keypairs should be held by core contributors only. If you're a non-core contirbutor going
+            through these steps, the PR process will facilitate a keypair holder being picked. That
+            person will generate the keypair, provide pubkey for PR, and ultimately enable the feature.
+
+        2. Add a public module for the feature, specifying keypair pubkey as the id with
+        `solana_sdk::declare_id!()` within the module. Additionally, add an entry to `FEATURE_NAMES` map.
+
+        3. Add desired logic to check for and switch on feature availability.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: Describe why the new feature gate is needed and any necessary conditions for its activation
+    validations:
+      required: true
+  - type: input
+    id: id
+    attributes:
+      label: Feature ID
+      description: The public key of the feature account
+    validations:
+      required: true
+  - type: dropdown
+    id: activation-method
+    attributes:
+      label: Activation Method
+      options:
+        - Single Core Contributor
+        - Staked Validator Vote
+    validations:
+      required: true
+  - type: input
+    id: testnet
+    attributes:
+      label: Testnet Activation Epoch
+      placeholder: Leave empty if not yet activated
```

Comment:
For these activation epochs, won't all new feature-gates start off not-yet-activated? If yes, these fields would all be empty to begin, yes? Does this imply that when folks activate features they should come back to the Issue and retroactively fill in these fields?

Whether yes or no to the above questions, maybe some additional clarification can be added here? (Since I haven't activated a feature yet, I'm personally not sure what the expectations/answers are for the above questions 😄 ).

Reply:
Sounds good, updated!